content/scalability/docs/supporting-infrastructure.md

```diff
@@ -0,0 +1,228 @@
+```
+
+Workloads being scheduled in a cluster with the Kubernetes Cluster Autoscaler should match a node selector to node groups based on label matching.
+
+```
+spec:
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key: eks.amazonaws.com/nodegroup
+            operator: In
+            values:
+            - 8-core-node-group    # match your node group name
+```
+
+### Use compute resources efficiently
+
+Compute resources include EC2 instances and availability zones. Using compute resources effectively will increase your scalability, availability, performance, and reduce your total cost. Predicting what instance types your cluster will need is an extremely difficult problem which is why we recommend not trying to predict every use case when you create a cluster. Instead of predicting and creating node groups you should use [Karpenter](https://karpenter.sh/) to provision workload instances on-demand based on the workload needs.
+
+Karpenter allows you to take advantage of all EKS compatible EC2 instance types as well as their different capacity types (e.g. on-demand, spot). This allows workloads to declare the type of compute resources it needs without first creating node groups or configuring label taints only for specific workloads. Please see the [Karpenter best practices](https://aws.github.io/aws-eks-best-practices/karpenter/) for more information. Consider enabling [consolidation](https://aws.github.io/aws-eks-best-practices/karpenter/#configure-requestslimits-for-all-non-cpu-resources-when-using-consolidation) in your Karpenter provisioner to replace nodes that are not fully utilized.
+
+Review the guide for setting up [highly-available applications](https://aws.github.io/aws-eks-best-practices/reliability/docs/application/) to make sure you have enough replicas to handle incidents.
+
+### Automate Amazon Machine Image (AMI) updates
+
+Keeping worker node components up to date will make sure you have the latest security patches and compatible features with the Kubernetes API. Updating the kublet is the most important component for Kubernetes functionality, but automating OS, kernel, and locally installed application patches will reduce maintenance as you scale.
+
+It is recommended that you use the latest [Amazon EKS optimized Amazon Linux 2](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami.html) or [Amazon EKS optimized Bottlerocket AMI](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami-bottlerocket.html) for your node image. Karpenter will automatically use the [latest available AMI](https://karpenter.sh/docs/concepts/provisioners/#instance-types) to provision new nodes in the cluster. Managed node groups will update the AMI during a [node group update](https://docs.aws.amazon.com/eks/latest/userguide/update-managed-node-group.html) but will not update the AMI ID at node provisioning time.
```

Comment:
Can you add an example for MNGs to make it clearer to the reader when the AMI is updated?

Reply:
Ah, I see you address this in the following paragraph.